client/signup/steps/p2-join-workspace/index.jsx

```diff
@@ -46,9 +49,23 @@ function P2JoinWorkspace( { flowName, goToNextStep, positionInFlow, stepName, su
 	}, [ userEmail ] );
 
 	useEffect( () => {
+		debug( 'Fetching workspace list' );
 		fetchList();
 	}, [ fetchList ] );
 
+	useEffect( () => {
+		if ( eligibleWorkspaces.length > 0 || isLoading ) {
+			return;
+		}
+
+		submitSignupStep( {
+			stepName,
+			wasSkipped: true,
```

Comment:
are we going to be able to differentiate skipped from not skipped in Tracks?

Reply:
Turns out we won't, so I added specific tracks for these, and a bunch of other ones too.